src/d3a/d3a_core/redis_connections/redis_area_market_communicator.py

```diff
@@ -116,7 +121,7 @@ def sub_to_multiple_channels(self, channel_callback_dict):
     def start_communication(self):
         if not self.is_enabled:
             return
-        if not self.pubsub.channels:
+        if not self.is_subscribed_to_channels_patterns:
```

Comment:
Redis pubsub already provides the same functionality: https://github.com/redis/redis-py/blob/5cd878de3eca3322d6719822d4999595f943b218/redis/client.py#L1205-L1208

I'd suggest removing the `is_subscribed_to_channels_patterns` property and use the one provided by the library.

Reply:
Thanks! Did not know that pubsub already provided this, awesome